rDNA/R/rDNA.R

```diff
@@ -1069,9 +1064,7 @@ dna_colorStatementType <- function(connection, statementType, color = "#FFFF00")
   if (is.null(color) || is.na(color) || length(color) != 1 || !is.character(color)) {
```

Comment:
As the `!is.character(color)` check is kept here, the color must be provided as a character string. What happens if the user provides the color as an integer, for example, as is common in R plotting? Is there a way to take this into account in the function and remaining code? Or should we just ignore this possibility?

Reply:
I wasn't aware integer colors are possible. But `rDNA:::col2hex()` plays nicely with them so there is no reason not to allow the user to use integers.